lib/install/management/create.go

```diff
@@ -229,6 +220,9 @@ func (d *Dispatcher) cleanupAfterCreationFailed(conf *config.VirtualContainerHos
 			d.op.Debug("Successfully cleaned up dangling bridge network.")
 		}
 	}
+
+	// delete the folder -- this will ONLY delete a folder if vic created it
```

Comment:
> this will ONLY delete a folder if vic created it

I don't see what part of the implementation of the method being called guarantees this behavior.

Reply:
This is during a create failure -- the only time that the `session.VCHFolder` will differ from `session.VMFolder` will be when `vic-machine` creates the folder.  We can update the comment to state such...

Reply:
To be clear -- the comment is directed _only_ at create...it is possible that we could delete an empty folder we didn't create -- that would happen if a user would relocate the appliance into a folder they created.

Reply:
I've updated the comment in...yep you guessed it: #7690

Reply:
No need to update the comment; now that I read this method again it makes more sense.